iib/workers/tasks/build_merge_index_image.py

```diff
@@ -279,8 +279,11 @@ def handle_merge_request(
         log.info('Getting bundles present in the source index image')
 
         with set_registry_token(overwrite_target_index_token, target_index, append=True):
+            source_bundles_dir = os.path.join(temp_dir, 'source_bundles')
```

Comment:
I am thinking, do we work with the database later on? 
Can we save some time/space when we keep this in directly in tmp folder? The think is we add bundles which are missing in source. So it means we would need to extract the source again? Was it previously done again - or  have we used the already extracted config?

I did not inspect the code that deeply yet. This is just an idea.